### DISCUSSION TITLE
post/20/

### DISCUSSION THREAD

**Discussion by giscus** [HoshinoRei/blog#10] [General]
# post/20/

一直以来，小玲都是用 Scoop 来安装 Node.js 的。以前小玲只用 main bucket 下的 nodejs 这个包。因为小玲以前没有使用旧版的 Node.js 的需求（虽然这个问题也能用 Scoop 的 version bucket 解决。），有魔

https://blog.bling.moe/post/20/

**Comment by Wanderingkang** [comment]
是不是删了好几篇文章

**Reply by HoshinoRei** [reply]
好几篇不至于。
